src/api/import.js

```diff
@@ -96,13 +81,8 @@ export default async (providedOptions = {}) => {
     'electron-winstaller': 'already uses this module as a transitive dependency',
   };
 
-  let electronName;
   for (const key of keys) {
-    if (key === 'electron' || key === 'electron-prebuilt') {
-      delete packageJSON.dependencies[key];
-      delete packageJSON.devDependencies[key];
-      electronName = key;
-    } else if (buildToolPackages[key]) {
+    if (buildToolPackages[key]) {
```

Comment:
import should move `electron`(`-prebuilt`) out of `dependencies` and into `devDependencies`.

Reply:
That seems like a separate concern, should do it in a separate PR 👍